README.md

```diff
@@ -54,11 +54,44 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 for trello integration 
 the following variables/section  are required in the projects .env file
 
-#Trello Keys
+## Trello Keys
 boardid = ID of the Trello board being used
 trello_key = valid api key
 trello_token = valid api token
 listtodo = list id for the "Todo" list
 listdoing = listid for the "Doing" list
 listdone = listid for the "Done" List
 
+## Testing
+Using Pytest for automated testing.
```

Comment:
It would be good to also mention how a user can run these tests, e.g. with `poetry run pytest`